scripts/local-cluster/README.md

```diff
@@ -0,0 +1,20 @@
+### K3s Cluster
+
+This directory contains scripts and manifests to deploy a local, lightweight cluster based on [k3s](https://github.com/rancher/k3s).
+
+#### Requirements
+
+1. Docker
+1. kubectl
+1. helm
+
+#### Running
```

Comment:
this falls short on explaining what to run step-by-step and requires a deep inspection at the Makefile to find out - we want something that allows us to not keep taking a look at the actual targets each time we want to use this.

Reply:
Yes, noted I'll add some docs here

Reply:
resolved below